src/09_blog_topic_migration_v2.ts

```diff
@@ -0,0 +1,152 @@
+import { MigrationModule } from "@kontent-ai/cli";
+import { ManagementClient } from "@kontent-ai/management-sdk";
+import { BlogPostModel, BlogTopicModel } from "./models";
+import { contentTypes } from "./models/project/contentTypes";
+import KontentService from "./services/KontentService";
+import { getElementsParamCodename } from "./utils/kontentUtils";
+
+const PUBLISHED = "published";
+const ARCHIVED = "archived_32a589a";
+const ARCHIVED_2 = "archived";
+
+const getAllBlogPosts = async (): Promise<BlogPostModel[]> => {
+	try {
+		const res = await KontentService.Instance(true)
+			.deliveryClient.items<BlogPostModel>()
+			.type(contentTypes.blog_post.codename)
+			.emptyFilter(
+				getElementsParamCodename(
+					contentTypes.blog_post.elements.topic_c2ebd37.codename
+				)
+			)
+			.notEmptyFilter(
+				getElementsParamCodename(contentTypes.blog_post.elements.topic.codename)
+			)
+			.toPromise();
+
+		return res.data.items;
+	} catch (error) {
+		console.error(error);
+	}
+};
+
+const getAllTopics = async (): Promise<BlogTopicModel[]> => {
+	try {
+		const res = await KontentService.Instance(true)
+			.deliveryClient.items<BlogTopicModel>()
+			.type(contentTypes.blog_topic.codename)
+			.toPromise();
+
+		return res.data.items;
+	} catch (error) {
+		console.error(error);
+	}
+};
+
+const upsertBlogPostLangVariant = async (
+	apiClient: ManagementClient,
+	blogPost: BlogPostModel,
+	blogTopic: BlogTopicModel,
+	topicCodename: string
+) => {
+	try {
+		return await apiClient
```

Comment:
@ondrabus the issue seems to be here - the parameters seem to be correct but the `apiClient` call does not fire. Not sure why this is happening. Maybe further config of the management API client is needed?

Reply:
Issue resolved